.kokoro/publish-docs.sh

```diff
@@ -20,26 +20,39 @@ export PYTHONUNBUFFERED=1
 
 export PATH="${HOME}/.local/bin:${PATH}"
 
-# Install nox
-python3 -m pip install --user --upgrade --quiet nox
-python3 -m nox --version
-
 # build docs
 nox -s docs
 
-python3 -m pip install --user gcp-docuploader
-
 # create metadata
-python3 -m docuploader create-metadata \
+python3.10 -m docuploader create-metadata \
```

Comment:
Where is `docuploader` being installed now? I see it uses to be `pip install`ed in this file.

Reply:
This script was moved internally and is no longer included in this PR.